well-known/chains.go

```diff
@@ -0,0 +1,297 @@
+package wellknown
+
+import (
+	pbfirehose "github.com/streamingfast/pbgo/sf/firehose/v2"
+)
+
+type WellKnownProtocol struct {
+	Name          string
+	BlockType     string
+	BufBuildURL   string
+	BytesEncoding pbfirehose.InfoResponse_BlockIdEncoding
+	KnownChains   []*Chain
+}
+
+type Chain struct {
+	Name    string
```

Comment:
Is this the canonical network ID of The Graph? If yes, we should document the field.